src/shared/config/apiUrls/apiUrls.ts

```diff
@@ -0,0 +1,17 @@
+import * as configure from './configure';
```

Comment:
Have you thought about naming this file `index.ts`? That way an import of this file would just be ...../apiUrls instead of ....../apiUrls/apiUrls
That being said, it's node module resolution logic, that is handled through tooling (Webpack/TypeScript) and we're going further than the standard browser module resolution logic that expects full file names (with extensions)

Reply:
I did consider this but my thinking was to keep the same `apiURls.ts` file name throughout the codebase (some namespaces apiUrls are just files directly under /config).